### PR TITLE
LB-767: stop 'sanitising' docker compose container names

### PR DESCRIPTION
**Pull request by amCap1712**
LB-767

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -226,9 +216,6 @@ if [ "$1" == "spark" ]; then
 fi
 
 if [ "$1" == "int" ]; then
-    # Project name is sanitized by Compose, so we need to do the same thing.
-    # See https://github.com/docker/compose/issues/2119.
-    INT_COMPOSE_PROJECT_NAME=$(echo $INT_COMPOSE_PROJECT_NAME_ORIGINAL | awk '{print tolower($0)}' | sed 's/[^a-z0-9]*//g')
     INT_TEST_CONTAINER_NAME=listenbrainz
     TEST_CONTAINER_REF="${INT_COMPOSE_PROJECT_NAME}_${INT_TEST_CONTAINER_NAME}_1"
```
Comment (alastair) [inline test.sh:220]: I don't see TEST_CONTAINER_REF used anywhere in the file, can we clear this up while we're at it? Maybe there are some other variables that aren't used either

**Review by alastair — APPROVED** [review]
Looks good. A few unused variables that we can get rid of.
Because the project name will change, this will have the effect of leaving behind some unused images and volumes, but it should only happen during tests, so I think it's fine to just leave it as-is. Developers can docker prune in their own time to remove them.

File: msb-develop.sh [hunk 2]
```diff
@@ -9,13 +9,7 @@ fi
 
 
 COMPOSE_FILE_LOC=docker/docker-compose.yml
```
Comment (alastair) [inline msb-develop.sh:11]: this is the wrong compose file for messybrainz. In fact, I can't even see a compose file any more, I suspect it disappeared when the repos were merged. Is there a reason to start up messybrainz as a standalone server?

I recommend that we delete this file entirely, and optionally add back docker-compose.yml into the messybrainz/ directory. There's no reason to have a develop.sh file if docker-compose.yml is named the default name and in a directory that can be automatically used as the project name

File: msb-test.sh [hunk 3]
```diff
@@ -8,11 +8,8 @@
 # ./test.sh -d             clean test containers
```
Comment (alastair) [inline msb-test.sh:8]: not needed? Do msb tests get run while running regular test.sh?

**Review by alastair — COMMENTED** [review]
Some more cleanups that we can make

File: msb-develop.sh (hunk 2 above)
Comment (amCap1712) [reply msb-develop.sh:11]: Indeed, this is wrong. I am not sure whether we need messybrainz as separate server. Maybe @mayhem has some thoughts about it?

File: msb-test.sh (hunk 3 above)
Comment (amCap1712) [reply msb-test.sh:8]: MSB tests do run when running test.sh but msb-test.sh is not used anywhere.

Comment (amCap1712) [reply msb-test.sh]: Removed.

File: msb-develop.sh (hunk 2 above)
Comment (amCap1712) [reply msb-develop.sh]: Removed. It is still possible to run a local development server using `msb_manage.py` in case needed.

Comment (alastair) [reply msb-develop.sh]: MessyBrainz used to run as an external server and LB connected over http to submit data, but this no longer happens. I'm happy to remove this functionality and add it back later if we need it.

Comment (amCap1712) [reply msb-develop.sh]: Removed that as well.

**Review by alastair — APPROVED** [review]
tests work for me, looks good.

**Pull request merged**